terraform/team-members-datadog/_terraform.tf

```diff
@@ -0,0 +1,22 @@
+// Configuration for Terraform itself.
+
+terraform {
+  required_version = "~> 1"
```

Comment:
Isn't it a better idea to pin it to something before the BUSL license was introduced? e.g. `~> 1.5`

Reply:
AFAIK the license change doesn't impact us as end-users of Terraform, since we're not embedding Terraform or its source code in a commercial product that we're selling. At least that's how I understand their [FAQ](https://www.hashicorp.com/license-faq#implications-of-change-for-users).